overlord/snapstate/conflict.go

```diff
@@ -180,6 +180,15 @@ func checkChangeConflictExclusiveKinds(st *state.State, newExclusiveChangeKind,
 				ChangeKind: "create-recovery-system",
 				ChangeID:   chg.ID(),
 			}
+		case "remove-recovery-system":
```

Comment:
this is probably ok as it is a rare operation, OTOH it's a bit overkill, we might want to leave a TODO about creating finer grained exclusivity rules